edx-platform/pearson-regis-theme/lms/static/sass/partials/lms/theme/_dashboard.scss

```
@@ -0,0 +1,463 @@
+// Change cards from landscape to portrait
+
+.view-dashboard div.item .items.course.single-course {
+  float: left;
+  margin: 1%;
+  min-width: 250px;
+  padding: 0px;
```

```suggestion
  padding: 0;
```

I would suggest to remove the px unit to maintain consistency